admin_manual/installation/nginx_examples.rst

```diff
@@ -171,14 +171,14 @@ your nginx installation.
 ownCloud in a subdir of nginx
 =============================
 
-The following config should be used when ownCloud is placed within a subdir of 
-your nginx installation.
+The following config should be used when ownCloud is not in your webroot but placed under a different contextroot of 
+your nginx installation such as /owncloud or /cloud. The following configuration assumes it is placed under ``/owncloud``.
 
 ::
 
   upstream php-handler {
       server 127.0.0.1:9000;
-      #server unix:/var/run/php5-fpm.sock;
+      #server unix:/var/run/php7-fpm.sock;
```

Comment:
Well, it is in so far related as I ran into all the issues after upgrading from Ubuntu 14.04 to 16.04 which implicitly upgraded php5.5 to php7. Therefore I wanted to state clearly, that the above config is running with php7-fpm - which might be relevant for others troubleshooting their installation.